solutions/LeadsLOBSolution/package.json

```diff
@@ -11,23 +11,25 @@
     "test": "gulp test"
   },
   "dependencies": {
-    "@microsoft/sp-core-library": "1.7.0-plusbeta",
-    "@microsoft/sp-lodash-subset": "1.7.0-plusbeta",
-    "@microsoft/sp-office-ui-fabric-core": "1.7.0-plusbeta",
-    "@microsoft/sp-webpart-base": "1.7.0-plusbeta",
+    "@microsoft/sp-core-library": "1.8.0",
+    "@microsoft/sp-http": "1.8.0",
+    "@microsoft/sp-lodash-subset": "1.8.0",
+    "@microsoft/sp-office-ui-fabric-core": "1.8.0",
+    "@microsoft/sp-webpart-base": "1.8.0",
     "@pnp/spfx-controls-react": "^1.10.0",
     "@types/es6-promise": "0.0.33",
     "@types/react": "16.4.2",
     "@types/react-dom": "16.0.5",
     "@types/webpack-env": "1.13.1",
-    "react": "16.3.2",
-    "react-dom": "16.3.2"
+    "office-ui-fabric-react": "6.143.0",
+    "react-dom": "16.7.0",
+    "react": "16.7.0"
   },
   "devDependencies": {
-    "@microsoft/sp-build-web": "1.7.0-plusbeta",
-    "@microsoft/sp-tslint-rules": "1.7.0-plusbeta",
-    "@microsoft/sp-module-interfaces": "1.7.0-plusbeta",
-    "@microsoft/sp-webpart-workbench": "1.7.0-plusbeta",
```

Comment:
Shouldn't we have a reference to the rush-stack-compiler-2.9 (for example) and the updated tsconfig.json file.

Reply:
Good catch!